snapcraft/commands/remote.py

```diff
@@ -101,6 +107,13 @@ def fill_parser(self, parser: argparse.ArgumentParser) -> None:
             action="store_true",
             help="acknowledge that uploaded code will be publicly available.",
         )
+        parser.add_argument(
+            "--launchpad-timeout",
+            type=int,
```

Comment:
I'd love to allow something more human-readable here (e.g. "5h40m10s").

(Not requesting that change here though as it's probably out of scope, but it'd be a nice quality of life improvement for later)

Reply:
Agreed!  I'm sure there's a great library out there for this type of parsing.  Especially since most timeouts would be between 30 minutes and 6 hours.